docs/TPMmodulePinout.md

```diff
@@ -0,0 +1,100 @@
+## TPM modules pinout
+This chapter contains a few different variants of the TPM module pinout. 
+This research was made to design a few variants of TPM modules with the most 
+popular pinouts. In this chapter below, the same pinouts of TPM modules were 
+grouped.
+Research is based on the best sellers positions form 
+[Newegg.com](https://www.newegg.com/) supplier.
+
+## 1. AsRock H170M PRO4 - LPC
+Interface: LPC</br>
+[User Manual H170M](https://download.asrock.com/Manual/H170M%20Pro4.pdf)
+![Flash](images/pinouts/Asrock-H170M_PRO4.svg)
+
+## 2. AsRock Rack X470 D4U, ROMED8-2T Server Motherboards, EPC621D8A
+Interface: LPC</br>
+[User Manual X470](https://download.asrock.com/Manual/X470D4U.pdf) </br>
+[User Manual ROMED8-2T](https://download.asrock.com/Manual/ROMED8-2T.pdf) </br>
+[User Manual EPC621D8A](https://download.asrock.com/Manual/EPC621D8A.pdf)
+![Flash](images/pinouts/AsRock_Rack_X470_D4U_Server.svg)
+
+## 3. Asus M5A99FX Pro r2.0, MAXIMUS VII HERO, Z87 PLUS, MAXIMUS VII RANGER
+Interface: LPC</br>
+[User Manual M5A99FX](https://images10.newegg.com/UploadFilesForNewegg/itemintelligence/ASUS/E7472_M5A99FX_PRO_R21401484751352.pdf) </br>
+[User Manual MAXIMUS VII HERO](http://dlcdnet.asus.com/pub/ASUS/mb/LGA1150/MAXIMUS-VII-HERO/E9192_Maximus_Vii_Hero.pdf) </br>
+[User Manual Z87 PLUS](https://images10.newegg.com/UploadFilesForNewegg/itemintelligence/ASUS/E7831_Z87_PLUS1403687574304.pdf) </br>
+[User Manual MAXIMUS VII RANGER](https://dlcdnets.asus.com/pub/ASUS/mb/LGA1150/MAXIMUS-VII-RANGER/E9798_maximus_vii_ranger_ug_v2_WEB.pdf)
+![Flash](images/pinouts/Asus-M5A99FX_Pro_r2.0.svg)
+
+## 4. Asus MAXIMUS IX FORMULA, CROSSHAIR VIII DARK HERO, ROG STRIX B450-F Gaming
+Interface: LPC</br>
+[User Manual MAXIMUS IX FORMULA](http://dlcdnet.asus.com/pub/ASUS/mb/LGA1151/MAXIMUS_IX_FORMULA/E12314_MAXIMUS_IX_FORMULA_UM_V3_WEB.pdf)</br>
+[User Manual ROG CROSSHAIR](https://media.s-bol.com/Bnx1GRw6DXXo/original.pdf)</br>
+[User Manual B450-F](https://www.bhphotovideo.com/lit_files/403638.pdf)
+![Flash](images/pinouts/Asus-MAXIMUS_IX_FORMULA.svg)
+
+## 5. Asus TPM-M R2.0, Z97 PRO GAMER
```

Comment:
@piotrdanio Are you sure it is not compatible with the `6. Asus-ROG STRIX X570-F GAMING` ?

Reply:
ok